core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c

```diff
@@ -206,7 +206,7 @@ wasi_clock_time_get(wasm_exec_env_t exec_env,
     if (!validate_native_addr(time, sizeof(wasi_timestamp_t)))
         return (wasi_errno_t)-1;
 
-    return wasmtime_ssp_clock_time_get(clock_id, precision, time);
+    return os_clock_time_get(clock_id, precision, time);
```

Comment:
same here, please revert this change
```suggestion
    return wasmtime_ssp_clock_time_get(clock_id, precision, time);
```